httpserver-rs/settings.md

```diff
@@ -27,7 +27,7 @@ An empty tls section, or no tls section, disables tls. To enable TLS, both `cert
 - `allowed_headers` - a list of allowed headers, case-insensitive. See [`Access-Control-Allow-Headers`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers) Default:
```

Comment:
GH isn't letting me comment on the file, but I think we need a top-level entry for `cache_control` now, right? These aren't intended to be set as part of the `cors` link def value, or are they?